app/scripts/shell/dimSearchFilter.directive.js

```diff
@@ -237,6 +240,44 @@
       },
       'keyword': function(predicate, item){
         return !!~item.name.toLowerCase().indexOf(predicate);
+      },
+      'light': function(predicate, item){
+        if (predicate.length === 0 || item.primStat == undefined) return false;
+        if (item.primStat.statHash != '3897883278' && item.primStat.statHash != '368428387') return false;
```

Comment:
why? because they were classified items? 

if that IS the case, then should we instead use `if(item.classified) return false;`? classified (true) is a property applied to all ... classified items.

unless there is something i'm forgetting...?